src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -138,6 +161,84 @@ Dictionary
               </label>
             </div>
             <br />
+
+
+            <div id="other-search">
+              <div className="form-check">
+                <input
+                  className="form-check-input"
+                  type="radio"
+                  name="exampleRadios"
+                  id="otherSourcesOption"
+                  onClick={this.otherSourceClick}
+                  value="option3"
```

Comment:
Can we get a better value other than option3?

Reply:
I have removed 'value'. it is not needed